.env.example

```diff
@@ -74,3 +74,13 @@ PAID_PLAN_PRICE=
 CACHE_DRIVER=database
 SESSION_DRIVER=file
 QUEUE_DRIVER=sync
+
+#Default fileststem to update contact images too
```

Comment:
It's a small detail but can you add an ` ` between `#Default` so it becomes `# Default`? Same for line 79.